Subject: use ioctl.h instead of ttycom.h on Linux
From: Robert Schwebel <r.schwebel@pengutronix.de>

When compiling nfs-utils-1.1.6, I get this error:

arm-iwmmx-linux-gnueabi-gcc -DHAVE_CONFIG_H -I. -I../../support/include -isystem /home/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/sysroot-target/include -isystem /home/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/sysroot-target/usr/include -D_GNU_SOURCE -Wall -Wstrict-prototypes  -pipe -g -O2 -MT sm_inter_svc.o -MD -MP -MF .deps/sm_inter_svc.Tpo -c -o sm_inter_svc.o sm_inter_svc.c
sm_inter_svc.c:10:39: error: sys/ttycom.h: No such file or directory

Use sys/ioctl.h instead.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
# wsa: 20091230: applied upstream \o/ : http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commitdiff;h=e456ee18bfbe422750dfad36f0afe2583f412538

 tools/rpcgen/rpc_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: nfs-utils-1.1.6/tools/rpcgen/rpc_main.c
===================================================================
--- nfs-utils-1.1.6.orig/tools/rpcgen/rpc_main.c
+++ nfs-utils-1.1.6/tools/rpcgen/rpc_main.c
@@ -550,7 +550,7 @@ s_output(int argc, char **argv, char *in
 	  f_print(fout, "#include <sys/ttycom.h>/* TIOCNOTTY */\n");
 #else
 	if( !tirpcflag )
-	  f_print(fout, "#include <sys/ttycom.h>/* TIOCNOTTY */\n");
+	  f_print(fout, "#include <sys/ioctl.h>/* TIOCNOTTY */\n");
 #endif
 	if( Cflag && (inetdflag || pmflag ) ) {
 	  f_print(fout, "#ifdef __cplusplus\n");
